ethereum/core/src/test/java/tech/pegasys/pantheon/ethereum/privacy/PrivateTransactionTest.java

```diff
@@ -50,6 +50,14 @@
           + "6e766966746a69697a706a52742b4854754642733d8a726573747269637465"
           + "64";
 
+  private static final String VALID_PRIVATE_TRANSACTION_RLP_PRIVACY_GROUP =
```

Comment:
Can this be generated so that it is clear how this is built?

Reply:
No - because I am testing here that the generation will produce these fixed values. If I generate these values and test if the generator produces the generated values (which it obviously will) I am not really testing anything. If someone messes with the generator then we will not catch the error :)